### ISSUE TITLE
Can we change the SimpleLogin license?

### ISSUE THREAD

**Issue by therealmrmuffin** [simple-login/app#1211]
I suggest the SimpleLogin license gets change to the "Attribution-NonCommercial-ShareAlike 4.0 International" license. Seen as i remember being told by @nguyenkims that SimpleLogin wasn't really meant to be used for commercial use (chance the "Non Commercial" version) if you self host it. I did take a pretty long time to think of the perfect license for this, and i think the "Attribution-NonCommercial-ShareAlike 4.0 International" license was really fitting for SimpleLogin. Creative commons is a well respected non profit organization who makes respectable licenses. I have chosen the "Attribution-NonCommercial-ShareAlike 4.0 International" license, because of the fact that it easy to understand (with their quick descriptions) and because of the fact they have a whole list of licenses to pick from.

Short Version: https://creativecommons.org/licenses/by-nc-sa/4.0/
Long Version: https://creativecommons.org/licenses/by-nc-sa/4.0/legalcode

**Comment by akash07k** [comment]
Yes, great idea.
I hope the license gets changed soon.
what about AGPL v3?

**Comment by therealmrmuffin** [comment]
> Yes, great idea. I hope the license gets changed soon. what about AGPL v3?

@akash07k That's up to them. I made my recommendation for what license they should change to, and now i am just waiting to hear back from them.

**Comment by alpha-tango-kilo** [comment]
> I suggest the SimpleLogin license gets change to the "Attribution-NonCommercial-ShareAlike 4.0 International" license.

Short answer: Creative Commons does not recommend its usage for code/software.

https://creativecommons.org/faq/#can-i-apply-a-creative-commons-license-to-software

Further comparison:

AGPL v3 (which this repository already uses) is more 'freedom friendly' / 'copyleft' than CC and so is what I would expect most users - who are interested in keeping SL open and libre - to want. It's basically best-in-class (subjective). Plus, due to the level of transparency AGPL requires (source code available, even on derivative works), it often completely discourages companies using it for commercial purposes (without obtaining a separate licensing agreement from SimpleLogin, which CC wouldn't prevent).

Please note I've never read either license in full, but that's my overall jist/comparison.

TL;DR: I don't think anything should change, and based on the points you've raised @therealmrmuffin, you should be happy with the existing AGPL license.

**Comment by akash07k** [comment]
@alpha-tango-kilo  In my opinion AGPL V3 is best.
Even I too use it a lot it for my opensource projects.
